### ISSUE TITLE
ASP.NET Core endpoint definition comments 

### ISSUE THREAD

**Issue by Rick-Anderson** [dotnet/AspNetCore.Docs#24382]
In the 2nd snippet under https://docs.microsoft.com/en-us/aspnet/core/fundamentals/routing?view=aspnetcore-3.1#aspnet-core-endpoint-definition-1, there are comments on each endpoint. The comments were removed in #23932

@serpent5  Should they be restored?

![image](https://user-images.githubusercontent.com/3605364/146814285-dc50761a-69cf-4085-a9a7-71c3f65090af.png)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 32374077-e02e-d6a6-1cea-05e58447b7cd
* Version Independent ID: 86aeab13-1c25-9fc1-4e56-0390cb40c242
* Content: [Routing in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/routing?view=aspnetcore-6.0)
* Content Source: [aspnetcore/fundamentals/routing.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/fundamentals/routing.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by serpent5** [comment]
I don't think it would hurt to add them back. My thinking at the time was that the explanation that followed, along with the sample output, covered it all in detail. I've also been trying to move away from inline comments where possible, as I've seen we have the "should we translate comments" notice on some of them.

What brought this to your attention?

**Comment by Rick-Anderson** [comment]
> . I've also been trying to move away from inline comments where possible, as I've seen we have the "should we translate comments" notice on some of them.

I'm the one who started that survey. Many folks said, "Why do it survey, it's obvious they would like translated comments.". I was sure that was true but I needed some data to get the translation team interested. It turns out virtually no one wants MT comments.

I used to avoid them, but now when they're useful I add them. 

> What brought this to your attention?

I stumbled across it and it seemed better with the comments.
